go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -1311,6 +1311,26 @@ func TestDoltVerifyConstraints(t *testing.T) {
 	}
 }
 
+func TestDoltStorageFormat(t *testing.T) {
+	var expectedFormatString string
+	if types.IsFormat_DOLT(types.Format_Default) {
+		expectedFormatString = "NEW ( __DOLT__ )"
+	} else {
+		expectedFormatString = "OLD ( __LD_1__ )"
+	}
+	script := queries.ScriptTest{
+		Name:        "dolt storage format function works",
+		SetUpScript: []string{},
```

Comment:
(super minor) You could completely leave this line out if you wanna save a line of code 😉